src/components/home/index.css

```diff
@@ -0,0 +1,34 @@
+.relatedCardsContainer {
+  background-color: #c8ccd6;
+  padding: 5rem 10% 5rem 10%;
+}
+
+.card {
```

Comment:
the problem with doing `.card` here is that if any of your teammates want to use `.card` (the bootstrap class, i think), then they will also inherit these styles, so you need to add a more specific class to the card that you're using